test/e2e/events_test.go

```diff
@@ -116,4 +116,14 @@ var _ = Describe("Podman events", func() {
 		Expect(result.ExitCode()).To(BeZero())
 	})
 
+	It("podman events format", func() {
+		Skip("need to verify images have correct packages for journald")
```

Comment:
Unconditionally skipped?

Reply:
we should at least be able to do this if distro == fedora ... @QiWang19 there is a helper for this:
```
info := GetHostDistributionInfo()
info.Distribution
```